nb-extension/js/init/init.js

```diff
@@ -113,7 +113,11 @@ define([
 
         var bower_root = baseURL + 'urth_components';
 
+        // expose suppressErrors, false by default to display errors
+        window.Urth.suppressErrors = config && config.suppressErrors || false;
```

Comment:
@aluu317 You should not need that `|| false` at the end. `undefined` should coerce to `false` in downstream check. If you really want a boolean, then it is more elegant to do 
``` javascript
window.Urth.suppressErrors = !!(config && config.suppressErrors)
```